src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -41,6 +41,60 @@ tags:
       </request>
       ```
       ... to this endpoint: `POST /request?cmd=create`
+
+
+      ## Workflow 2: Add a Review
+      **Add a User as Reviewer of a Submit Request**
+
+      First we make sure that a certain user can review our submit request.
+      We will add the user `test1` as a reviewer of the already created submit request.
+
+      For the sake of simplicity we suppose, that:
+        - There is an already created user, with login name `test1`.
+        - There is an already created submit request, with number `33`.
+
+      Adding the user `test1` as a reviewer of the submit request number `33` can be performed with this command:
+      `osc review add -m 'Add user test1 as reviewer' -U test1 33`
+
+      The same action can be done with this API endpoint: `POST /request/34?cmd=addreview&by_user=test1`
+
+      **Review a Submit Request as Accepted**
+
+      Now that the user `test1` is a reviewer of the submit request number `33`, this user can review that submit request as `accepted`.
+
+      To review the submit request as `accepted`, issue the following command, using for authentication the login `test1`:
+      `osc review accept -m 'Accept submit request' -U test1 33`
+
+      This can also be done with this API endpoint, using for authentication the login `test1`:
+      `POST /request/33?cmd=changereviewstate&newstate=accepted&by_user=test1`
+
+
+      ## Workflow 3: Approve a Submit Request
+
+      To mark a submit request as approved means that a submit request will be automatically accepted when all the required reviews are accepted.
+
+      For the sake of simplicity let's accept that:
+        - There is an already created user, with login name `test1`.
```

We can combine this sentence with the next one:

```suggestion
        - There is already a user with the login name `test1`, which is also a maintainer of the project `home:Admin`.
```